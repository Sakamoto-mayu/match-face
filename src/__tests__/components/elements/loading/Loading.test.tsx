import Loading from "@/components/elements/loading/Loading";
import { cleanup, render } from "@testing-library/react";

describe("Loading.tsx", () => {
  beforeEach(() => {
    cleanup();
  });
  afterAll(() => {
    cleanup();
  });
  it("スナップショットテスト", () => {
    const { container } = render(<Loading />);
    expect(container).toMatchSnapshot();
  });
});
