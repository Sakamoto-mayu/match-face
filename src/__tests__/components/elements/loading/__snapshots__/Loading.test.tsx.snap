// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading.tsx スナップショットテスト 1`] = `
<div>
  <button
    class="border rounded-lg p-2 fixed bottom-10 left-10"
    disabled=""
    type="button"
  >
    <svg
      class="animate-spin h-12 w-12 flex justify-center"
      fill="none"
      viewBox="0 0 24 24"
    >
      <circle
        class="opacity-25"
        cx="12"
        cy="12"
        r="10"
        stroke="currentColor"
        stroke-width="4"
      />
      <path
        class="opacity-75"
        d="M4 12a8 8 0 018-8V0C5.373 0 0 5.373 0 12h4zm2 5.291A7.962 7.962 0 014 12H0c0 3.042 1.135 5.824 3 7.938l3-2.647z"
        fill="currentColor"
      />
    </svg>
  </button>
</div>
`;
